tokens/C23-1325d3/info.json

```diff
@@ -1,11 +1,14 @@
 {
 "website": "https://c23token.net",
 "description": "Conglomerate 23 Decentralized Autonomous Organization",
+"status": "active",
 "social": {
     "email": "support@23co.in",
     "blog": "https://c23token.net",
     "twitter": "https://twitter.com/c23token",
     "whitepaper": "https://c23token.net/whitepaper.pdf"
   },
-"status": "active"
+"lockedAccounts": {
+        "erd134hqafduq7kslfhrggsy9jhxu4cgu5m6tagkrjq630hgp204fd7szc5r3q": "distribution_sc"
```

Comment:
This address does not represent a smart contract address. 

This should be named differently.

Thanks

Reply:
is ok if i name it with herotag_distribution?
Example: c23dao_distribution
Thanks